app/build.gradle

```diff
@@ -241,7 +242,7 @@ android.applicationVariants.all { variant ->
 // -------------------------------------------------------------------------------------------------
 // Feature build flags
 // -------------------------------------------------------------------------------------------------
-    buildConfigField 'Boolean', 'SEND_TAB_ENABLED', (buildType == "nightly" || !isReleased).toString()
+    buildConfigField 'Boolean', 'SEND_TAB_ENABLED', (buildType == "nightly").toString()
```

Comment:
This seems that it would disable the send tab feature for developer debug builds, so it would be very hard to actually test and improve the feature.

Reply:
Good catch, I forgot to revert this from when playing with different `is_released`/`is_debug` options